### ISSUE TITLE
Random Hidden and Epochs > 1

### ISSUE THREAD

**Issue by Ageliss** [SafeAILab/EAGLE#152]
Hi EAGLE Team,

Thanks for your great work that accelerates speculative decoding up to unbelievable 4~5 times.
But I can't reproduce the result of below: Acc rate improves with more epochs.
<img width="886" alt="image" src="https://github.com/user-attachments/assets/c773443e-0d63-4895-9134-be6c49501d5f">

In our expriments, more than 1 epoch on sft data results in acc rate slightly drops. Was random data augmentation a key feature to avoid such observation??

**Comment by Liyuhui-12** [comment]
What training data did you use, and what is its size?

**Comment by Ageliss** [comment]
> What training data did you use, and what is its size?
We actually used our human annotated sft dataset, about 110k~240k. We also masked out the system and user prompt to train the human response only.
The size of ShareGPT the paper used is about 68k.
